config.ini

```diff
@@ -54,3 +54,10 @@ dem_type = Copernicus 30m Global DEM II
 
 # Temporarily changes GDAL_NUM_THREADS during processing. Will be reset after processing has finished.
 gdal_threads = 4
+
+# Extended Timing Annotation Dataset (ETAD) correction
+# https://sentinel.esa.int/web/sentinel/missions/sentinel-1/data-products/etad-dataset
+# If etad==True, etad_dir is searched for ETAD products matching the respective input SLC
+# and a new SLC is created in tmp_dir, which is then used for all other processing steps.
```

Comment:
Maybe also add "If etad==False, [etad_dir] will be ignored."

Reply:
Sure: https://github.com/SAR-ARD/S1_NRB/pull/14/commits/b410815ce0c9da7ccf78a09f112688c0355d8c89